user/disable-job-logs.md

```diff
@@ -0,0 +1,36 @@
+|------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------|
+| **Access to logs only for write/push users OFF, Access to old logs OFF (default settings)**        | Users with **read access** to the repository can see build job logs for public repo not older than the last 365 days.                                                                                                                                                                                                                           | Users with **read access** to the repository can see the build job logs not older than the last 365 days.   |
+| **Access to logs only for write/push users ON, Access to old logs OFF**                            | Users with **write access** to the repository will see the build job logs not older than the last 365 days.                                                                                                                                                                                                                                       | Users with **write access** to the repository will see the build job logs not older than the last 365 days. |
+| **Access to logs only for write/push users ON, Access to old logs ON**                             | Users with **write access** to the repository will see the build job logs, full history                                                                                                                                                                                                                                                     | Users with **write access** to the repository will see the build job logs, full history.              |
+| **Access to logs only for write/push users OFF,  Access to old logs ON(state of today's product)** | Users with **read access** to the repository see all build job logs and full history. In this case, even anonymous Users are treated as Users with read access because everybody can view the public repository in the original VCS. That collaboration pattern must still be available as an option to the repository owners. | Users with **read access** to the repository can see build job logs and full history.                    |
```

Comment:
@nrios14 Please remove the sentence: That collaboration pattern must still be available as an option to the repository owners.

Reply:
Obsolete. I already updated.